project4/project/ebs/views.py

```diff
@@ -251,4 +254,51 @@ def create_blog(request):
 
 @login_required(login_url='/')
 def manage_blog(request):
-    return render(request,'ebs/manage_blog.html')
\ No newline at end of file
+    try:
+        orgobj=Organisation.objects.get(user_id=request.user.id)   
+        data=list()
+        context={}
+        bloglist=Blog.objects.filter(organisation_id=orgobj.id).order_by('-timestamp')
+        for each in bloglist:
+            context={'blog_id':each.id,
+                'blog_title':each.title,
+                'blog_description':each.description
+            }
+            files=BlogFile.objects.filter(blog_id=each.id).order_by('-id')
+            if files is None:
+                context['blog_file']=''
+            else:
+                for each in files:
+                    print each.attachments
+                    filename, file_extension = os.path.splitext(str(each.attachments))
+                    if  file_extension in ['.png','.jpeg','.jpg']:
+                        context['blog_file']=each.attachments
+                        break
+
+            data.append(context)
+        paginator=Paginator(data,5)
```

Comment:
Keep the 5 (pagination count) above as a global variable in settings which is configurable